.github/workflows/build-desktop-release.yml

```diff
@@ -436,8 +436,15 @@ jobs:
 
   nightly-release:
     if: ${{ github.event_name == 'schedule' || github.event.inputs.build-target == 'nightly' }}
-    needs: [ build-macos-x64, build-macos-arm64, build-linux, build-windows, build-android ]
-    runs-on: ubuntu-18.04
+    needs:
+      [
```

Comment:
Are you using any auto formatter?
This is equivalent to 

```yaml
needs:
  - build-a
  - build-b
  - build-c
```

Reply:
@andelf reverted the style changes